agent/command.go

```diff
@@ -114,6 +115,33 @@ func (a *Agent) runCommandSet(ctx context.Context, tc *taskContext, commandInfo
 			tc.setCurrentIdleTimeout(nil)
 		}
 
+		if cmd.Name() == "manifest.load" {
```

Comment:
Rather than hardcoding command names, if we're going to be looking at them here can we add to globals? 
https://github.com/evergreen-ci/evergreen/blob/7d08b7af8de36133c90b0b67d785c1b14bb5c258/globals.go#L514